modules/deployment/src/test/resources/META-INF/mule-module.properties

```diff
@@ -1,3 +1,6 @@
 module.name=deployment-test
 
-artifact.export.classPackages=org.mule.runtime.module.deployment.internal
+artifact.export.classPackages=org.mule.runtime.module.deployment.internal,\
+                              org.mule.functional.api.component
```

Comment:
Deployment test must install the test plugin in the container instead of exposing the resources using this module. I'm OK to have this as a workaround, but create a jira to improve this